### PR TITLE
NXP-27973: replace ftests with non admin users

### PR DESCRIPTION
**Pull request by PGoncalvesNX**

### CODE REVIEW

**Comment by PGoncalvesNX** [comment]
https://qa.nuxeo.org/jenkins/view/TestAndPush/job/web-ui-pipeline-fdsantos/54/ 🎉

File: ftest/features/datatable_editing.feature [hunk 1]
```diff
@@ -19,6 +19,7 @@ Feature: Multi-valued Properties Editing
       | name                  | value                                |
       | multiString           | [["toto"],["pouet"],["foo"],["bar"]] |
 
+  #NXP-28141 is preventing this test from passing with a non-Admin user
```
Comment (nmpcunha) [inline ftest/features/datatable_editing.feature]: nit: a space between `#` and the comment itself would be nice and consistent on how we write comments

File: ftest/features/ordered_folder.feature [hunk 2]
```diff
@@ -2,6 +2,11 @@ Feature: Ordered Folder
 
   I can move documents up and down
 
+  Background:
+    Given user "John" exists in group "members"
+    And I login as "John"
+
+  #this test should fail. When NXP-27790 fails this should only pass with ReadWrite permission
```
Comment (nmpcunha) [inline ftest/features/ordered_folder.feature]: same nitpick as above

File: ftest/features/datatable_editing.feature (hunk 1 above)
Comment (Gabez0r) [reply ftest/features/datatable_editing.feature]: +1

File: ftest/features/user.feature [hunk 3]
```diff
@@ -4,8 +4,9 @@ Feature: User
 
   Background:
     Given user "jdoe" exists
-    Given I login as "Administrator"
-    And I click the "administration" button
+    Given user "John" exists in group "powerusers"
```
Comment (nmpcunha) [inline ftest/features/user.feature]: maybe `And user "John" exists..."

File: ftest/features/group.feature [hunk 4]
```diff
@@ -7,7 +7,8 @@ Feature: Group
       | name           | label    |
       | managers       | Managers |
       | managers/webui |          |
-    Given I login as "Administrator"
+    Given user "John" exists in group "powerusers"
```
Comment (Gabez0r) [inline ftest/features/group.feature]: I'd say that it is ok to keep using the Administrator when testing features that required administration rights.

Comment (PGoncalvesNX) [reply ftest/features/group.feature]: This change was coordinated with @tiagocardoso

File: ftest/features/ordered_folder.feature [hunk 5]
```diff
@@ -2,6 +2,11 @@ Feature: Ordered Folder
 
   I can move documents up and down
 
+  Background:
+    Given user "John" exists in group "members"
+    And I login as "John"
+
+  # this test should fail. When NXP-27790 fails this should only pass with ReadWrite permission
```
Comment (FabioFSantos) [inline ftest/features/ordered_folder.feature]: Don't you mean "When NXP-27790 is fixed" instead of fails

**Pull request merged**